src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DoNotPassLiteralsAsLocalizedParametersTests.cs

```diff
@@ -1281,10 +1323,25 @@ public void M1(C c)
     }}
 }}
 ",
-            // Test0.cs(14,13): warning CA1303: Method 'void Test.M1(C c)' passes a literal string as parameter 'param' of a call to 'void C.M(string param)'. Retrieve the following string(s) from a resource table instead: "a".
-            GetCSharpResultAt(14, 13, "void Test.M1(C c)", parameterName, $"void C.M(string {parameterName})", "a"));
+                    },
+                },
+            };
 
-            await VerifyVB.VerifyAnalyzerAsync($@"
+            if (useNameHeuristic)
```

Comment:
Ah, you are relying on the default value being `false` here? Works for me.

Reply:
Yes sorry I need to find a way to make it more explicit.